### ISSUE TITLE
Side-by-Side vs Overlay

### ISSUE THREAD

**Issue by drtjre** [4Quant/LungStageAnnotator#12]
Optional: Gregor would prefer a side-by-side view of the CT and PET volumes rather than an overlay, but can accept the overlay system for now.

{I understand this would probably be a major rework - so just reporting this here but do not expect it any time soon.}

**Comment by kmader** [comment]
you can now show both PET and CT independently but side-by-side might be trickier, it might be a good use for this fourth panel

**Comment by drtjre** [comment]
Good idea to set as "enhancement".  I think they can get used to using overlays instead- at least for now.
